lib/std/fmt.zig

```diff
@@ -32,9 +32,11 @@ pub const FormatOptions = struct {
 /// this format:
 /// `{[argument][specifier]:[fill][alignment][width].[precision]}`
 ///
-/// Each word between `[` and `]` is a parameter you have to replace with something:
+/// Above, each word including its surrounding [ and ] is a parameter which you have to replace with something:
```

Comment:
While we're clarifying things, I think "have to" is a poor choice of words when nearly all of these parameters can be omitted.

Reply:
Actually the full comment explains this clearly enough.